src/python/grpcio_status/grpc_status/rpc_status.py

```diff
@@ -62,7 +63,7 @@ def from_call(call):
     return None
 
 
-def to_status(status):
+def to_status(status: status_pb2.Status) -> _Status:
```

Comment:
`_Status` -> `grpc.Status`. `_Status` is an internal type. We need to use the public interface, not the internal implementation.